src/core/random.c

```diff
@@ -31,7 +42,7 @@ long int random ( void ) {
 	int32_t q;
 
 	if ( ! rnd_seed ) /* Initialize linear congruential generator */
-		srandom ( currticks() );
+		srandom ( 0 );
```

Comment:
That `s/currticks()/0/` feels wrong in a commit message titled _Add more entropy_.

Reply:
The idea was to make it obvious inside `srandom` whether we are given an exact seed value (for example from the tests), or if additional entropy needs to be added. So since `0` cannot be used as a seed, it seemed like the best approach. And, I figured it would make sense to keep all the entropy sources in one place -- unless there's a reason i'm missing to keep some of them separate :>